src/app/layout/header/components/brand-menu-dropdown/brand-menu-dropdown.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="btn-group" dropdown [autoClose]="false" container="body">
+  <button id="button-nested" dropdownToggle type="button" class="btn bg-white dropdown-toggle" aria-controls="dropdown-nested">
+    Shop By
+    <span *ngIf="taxonomies.length!==0">{{taxonomies[0].name}}</span>
```

Comment:
What if api only sends the array of size 1 only. Then will it not break at this point.

Can you please add an `*ngIf="taxonomies[0]; let brands_taxonomy"`